controllers/v1/solutionsController.js

```diff
@@ -2161,4 +1990,388 @@ module.exports = class Solutions extends Abstract {
       }
     });
   }
+
+  /**
+    * @api {post} /samiksha/v1/solutions/create Create solution
+    * @apiVersion 1.0.0
+    * @apiName Create solution
+    * @apiGroup Solutions
+    * @apiParamExample {json} Request-Body:
+    * {
+    * "resourceType" : [],
+    * "language" : [],
+    * "keywords" : [],
+    * "concepts" : [],
+    "themes" : [],
+    "flattenedThemes" : [],
+    "entities" : ["bc75cc99-9205-463e-a722-5326857838f8","8ac1efe9-0415-4313-89ef-884e1c8eee34"]
+    "registry" : [],
+    "isRubricDriven" : false,
+    "enableQuestionReadOut" : false,
+    "allowMultipleAssessemts" : false,
+    "isDeleted" : false,
+    "programExternalId" : "AMAN_TEST_123-1607937244986",
+    "entityType" : "school",
+    "type" : "improvementProject",
+    "subType" : "improvementProject",
+    "isReusable" : false,
+    "externalId" : "01c04166-a65e-4e92-a87b-a9e4194e771d-1607936956167",
+    "minNoOfSubmissionsRequired" : 2,
+    "allowMultipleAssessemts" : true
+    }
+    * @apiHeader {String} internal-access-token internal access token  
+    * @apiHeader {String} X-authenticated-user-token Authenticity token
+    * @apiSampleRequest /samiksha/v1/solutions/create
+    * @apiUse successBody
+    * @apiUse errorBody
+    * @apiParamExample {json} Response:
+    * {
+    "message": "Solution created successfully",
+    "status": 200,
+    "result": {
+        "_id": "5ff447e127ef425953bd8306"
+    }}
+    */
+
+  /**
+   * Create solution.
+   * @method
+   * @name create
+   * @param {Object} req - requested data.
+   * @param {String} req.params._id - solution id.
+   * @returns {JSON} Created solution data.
+   */
+
+  async create(req) {
+    return new Promise(async (resolve, reject) => {
+      try {
+        //passing {true} for checkDate params in helper
+        let solutionData = await solutionsHelper.createSolution(req.body, true , req.userDetails.userToken);
```

Comment:
@praveenKDass passing user token is required? Please check

Reply:
removed @VISHNUDAS-tunerlabs